config/options.go

```diff
@@ -385,16 +378,7 @@ func optionsFromViper(configFile string) (*Options, error) {
 // variables or from a file
 func (o *Options) parsePolicy() error {
 	var policies []Policy
-	// Parse from base64 env var
-	if o.PolicyEnv != "" {
-		policyBytes, err := base64.StdEncoding.DecodeString(o.PolicyEnv)
-		if err != nil {
-			return fmt.Errorf("could not decode POLICY env var: %w", err)
-		}
-		if err := yaml.Unmarshal(policyBytes, &policies); err != nil {
-			return fmt.Errorf("could not unmarshal policy yaml: %w", err)
-		}
-	} else if err := o.viperUnmarshalKey("policy", &policies); err != nil {
+	if err := o.viper.UnmarshalKey("policy", &policies, viperPolicyHooks); err != nil {
```

Comment:
I believe no one is using policy as a base64 blob anymore. That funcitonality has not been around (as far as I am aware) since v0.0.4.

Reply:
- this is part of integration test that was failing
- `POLICY_FILE` seems to be removed, yes 
- `POLICY` env is supported as per docs https://www.pomerium.io/reference/#policy